src/js/themes/base.js

```diff
@@ -693,6 +693,7 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       },
       control: {
         // extend: undefined,
+        // open: undefined,
```

Comment:
just a thought, not a must-have, but I personally like to add an example on one of the existing stories of Select that already has a custom theme, to be used as a POC of the new ability. let me know if that works.

Reply:
To confirm I understand -- your suggestion is to find an existing Select story, such as Basics.js, and demonstrate this styling capability by adding to the custom theme definition -- correct? If so, that makes sense and I would be happy to add.

Reply:
Yes exactly! This will help users understand how to customize the theme and see some of the changes i nthe storybook!

Reply:
Spot on @halocline & @oorestisime !

Reply:
Modified the "Custom" story to include styling for the open state of the control.

Reply:
I like it, looks great!